spring-security-compatibility/src/main/java/com/sap/cloud/security/comp/XsuaaTokenComp.java

```diff
@@ -1,368 +0,0 @@
-/**
- * SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Cloud Security Client Java contributors
- *
- * SPDX-License-Identifier: Apache-2.0
- */
-package com.sap.cloud.security.comp;
```

Comment:
I have second thoughts on removing this module. I think as we are keeping the `spring-xsuaa` it's still relevant to provide further support for easily switching to the new token interface. We should keep it as it is for time being.